assets/js/base/components/cart-checkout/order-summary/order-summary-item.js

```diff
@@ -121,11 +124,27 @@ const OrderSummaryItem = ( { cartItem } ) => {
 				<ProductImage image={ images.length ? images[ 0 ] : {} } />
 			</div>
 			<div className="wc-block-components-order-summary-item__description">
+				<span className="screen-reader-text" tabIndex={ 0 }>
+					{ sprintf(
+						/* translators: %d items of product %s */
+						__( '%1$d %2$s', 'woo-gutenberg-products-block' ),
+						quantity,
+						name
+					) }
+				</span>
 				<ProductName
 					disabled={ true }
 					name={ name }
 					permalink={ permalink }
 				/>
+
+				<span className="screen-reader-text" tabIndex={ 0 }>
+					{ sprintf(
+						/* translators: Unit Price %s */
+						__( 'Unit Price: %s', 'woo-gutenberg-products-block' ),
+						formatPrice( priceSingle, priceCurrency )
+					) }
+				</span>
```

Comment:
This is a bit tricky because when a product has a discount, we want to announce it to screen readers too. `ProductPrice` already has some logic to handle these cases.

Considering you already added screen reader text to the 'Total price', I wonder if we need to specify this is the unitary price. I think this will be clear now. How does it sound getting rid of this span and just leaving the 'total price' one?

Reply:
We can definitely remove this as it makes the screen reader easier to follow.